cognite/powerops/utils/serialization.py

```diff
@@ -18,9 +18,9 @@
 from yaml import CSafeLoader, safe_dump
 
 try:
-    import tomllib  # type: ignore[import]
+    import tomllib  # type: ignore[import-not-found, unused-ignore]

```

Comment:
**question**: What is the purpose of these unused-ignore here? Is mypy finding the import in one context but not in others (local/github workflow/...)

Reply:
Depending on which python version you're running mypy with it'll see this as a useless ignore, so just a bit of a weird case for library imports where it's version dependent which one is used